### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by parasjain-12**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,8 +142,35 @@ Plot all messages so far, with new messages each hour.
...
+ </details>
+ 
+ Compare the Friend request recieved received and friend request send.
+ ![](images/ComparingFriendRequestReceivedorSend.png)
+ 
+ <details>
+  <summary>We compare monthwise no.of friend request send vs friend request received</summary>
+  </details>
+ 
+## Observations
```
Comment (techytushar) [inline README.md]: These observations should be added in the observations section, no need to create a new section.

**Comment by techytushar** [comment]
@parasjain-12 nice work, but there was no need to make a new PR for this, these changes could have been added to your already opened PR. Please add the required images also.

**Comment by parasjain-12** [comment]
I tried to add it to already open PR but not able to do. I am contributing for the first time

**Comment by kaustubhhiware** [comment]
@parasjain-12 whoever told you you can make direct commit pushes to the repository?
You're supposed to make PR's for everything you want to change.
This behaviour will not be tolerated in the future. I'm reverting your changes for now, but any similar occurence would mean you would be removed as a collaborator from this project.

Obviously closing this PR.

**Comment by techytushar** [comment]
@parasjain-12 This is not the right way to contribute to a project. 
What you should do is first make a fork of the project, the clone it locally, then make a new branch, do all the changes you want, then add and commit all changes. Once this is done push the changes to you forked repo (origin) and then from there open a new PR.

**Pull request closed without merging**